content/build/buildkit/_index.md

```diff
@@ -105,9 +105,137 @@ If the `/etc/docker/daemon.json` file doesn't exist, create new file called
 `daemon.json` and then add the following to the file. And restart the Docker
 daemon.
 
+## BuildKit on Windows
+
 > **Warning**
 >
-> BuildKit only fully supports building Linux containers. Experimental Windows
-> container support is tracked in
-> [`moby/buildkit#616`](https://github.com/moby/buildkit/issues/616)
+> BuildKit only fully supports building Linux containers.
+> Windows container support is experimental, and is tracked in
+> [`moby/buildkit#616`](https://github.com/moby/buildkit/issues/616).
 { .warning }
+
+BuildKit has experimental support for Windows containers (WCOW) as of version 0.13.
+This section walks you through the steps for trying it out.
+We appreciate any feedback you submit by [opening an issue here](https://github.com/moby/buildkit/issues/new), especially `buildkitd.exe`.
+
+### Known limitations
+
+- BuildKit on Windows currently only supports the `containerd` worker.
+  Support for non-OCI workers is tracked in [moby/buildkit#4836](https://github.com/moby/buildkit/issues/4836).
+
+### Prerequisites
+
+- Architecture: `amd64`, `arm64` (binaries available but not officially tested yet).
+- Supported OS: Windows Server 2019, Windows Server 2022, Windows 11.
+- Base images: `ServerCore:ltsc2019`, `ServerCore:ltsc2022`, `NanoServer:ltsc2022`.
+  See the [compatibility map here](https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility?tabs=windows-server-2019%2Cwindows-11#windows-server-host-os-compatibility).
+- Docker Desktop version 4.29 or later
+
+### Steps
```

Comment:
nit: I notice you remove the step for enabling `Containers` Windows feature, was this intentional?

Reply:
No, thanks for catching that. I'll add it back in.